app/services/users/creator.rb

```diff
@@ -0,0 +1,25 @@
+class Users::Creator < Struct.new(:email)
+  def call
+    create_user
+    add_default_categories
+  end
+
+  private
+
+  def create_user
+    @user = User.create(
+      email: email,
+      password: generate_password,
+      confirmed_at: Date.current
+    )
+  end
+
+  def generate_password
+    Devise.friendly_token[0, 20]
+  end
+
+  def add_default_categories
+    Users::DefaultCategoriesCreator.new(@user).call
+    @user
```

Comment:
let's return `@user` from `#call` instead of `#add_default_categories`. `add_default_categories` doesn't say anything about a user, you can't expect that it returns `@user`

Reply:
@AleksSenkou Oh, sorry, I repair it later, forgot repair here.